src/components/Products/Filters.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
Every component should have it's own directory. Products, Filters, Price...

Reply:
Fixed. Thanks!

Reply:
No it is not)
![image](https://user-images.githubusercontent.com/69967761/197724991-09361b1a-010b-4499-a408-f2d252c3ec59.png)
It should be:
/src/components/Products/Products.js
/src/components/Filters/Filters.js
/src/components/Categories/Categories.js
...

Reply:
Thanks for clarifying, Vasyl! Now it's fixed.